app/controllers/studies_controller.rb

```diff
@@ -23,8 +23,8 @@ class StudiesController < ApplicationController
     check_access_settings
   end
   # special before_action to make sure FireCloud is available and pre-empt any calls when down
-  #before_action :check_firecloud_status, except: [:index, :do_upload, :resume_upload, :update_status,
-  #                                                :retrieve_wizard_upload, :parse]
+  before_action :check_firecloud_status, except: [:index, :do_upload, :resume_upload, :update_status,
+                                                  :retrieve_wizard_upload, :parse]
```

Comment:
were these commented out before for performance reasons?

Reply:
No - I think this was actually an error where we were testing something and forgot to uncomment it.  It most definitely _should_ be there as API outage will cause many of these methods to barf if we don't check.